### DISCUSSION TITLE
Finding Plugin User Presets When Using Audiogridder?

### DISCUSSION THREAD

**Discussion by Loupap** [apohl79/audiogridder#1276] [General]
I can't seem to find any of my user presets for a VST3 Plugin  when using Audiogridder. When I right click on the plugin insider of Audiogridder on the server side the word "Preset" is greyed out and nothing appers.

Can anyone help with this?

Do I need to reconsider a directory file path?

Thanks in Advance.

**Comment by apohl79** [comment]
There is two preset menus. The one you were looking at is showing plugin presets. The AG presets you are creating yourself can be found under the settings menu, top item.
